src/pages/product-list-page/product-list-filter/colors-filter/colors-filter.js

```diff
@@ -23,11 +23,12 @@ const ColorsFilter = () => {
 
   const { colorsFilter } = filters;
 
-  const colors = [
-    ...new Set(
-      filterData.map((product) => JSON.stringify(product.colors[0].simpleName))
-    )
-  ].map(JSON.parse);
+  const colors = filterData
+    .map((product) => product.colors[0].simpleName)
+    .filter(
```

Comment:
What are you trying to do here, it is hard for me to understand

Reply:
I use it to create filter checkbox values

Reply:
fixed